### ISSUE TITLE
Trying to reproduce the same result of paper.

### ISSUE THREAD

**Issue by akramayman** [MindAI/MiPepid#4]
Dear @MindAI, 

My partner at Saarland University and I tried to reproduce the same result of the paper using the same code I provided in the repository. We downloaded the dataset and took the same sample test data that was used in the test feature file [Sequence ID, Sequence] as a FASTA format,
We then ran MiPepid on this dataset and were surprised that we did not get the same results as they reported in the paper, we obtained about 77% overall accuracy, while for the negative data set it did not exceed 30% accuracy and we checked the K-mer results for the same sample of test data it was different than what provided in test feature file,  so we have a fear that we have done something wrong, or that some information is missing like did you use the same calculation of k-mer to generate the train and test features file, is it the same data, or we think that you exceed the Overfitting of data to got the 95%. 

One more note the code only works with the Scikit-learn version of 0.22 and It took us a long time to discover that.

It would be great to get information from you or help us understand. Thank you.

Best Regards,
Akram

![Screenshot_2](https://github.com/MindAI/MiPepid/assets/72864908/0430ea76-7946-441f-a892-469074c63b15)
